packages/categorize/src/categorize/index.jsx

```diff
@@ -116,6 +116,8 @@ export class Categorize extends React.Component {
     return flexDirection;
   };
 
+  existAlternateResponse = (correctResponse) => correctResponse?.some(correctRes => correctRes.alternateResponses?.length >0);
```

Comment:
Code looks good but spacing is a bit off here, maybe we can check if you have different a prettier setup.
There should always be spacing between operators.